test/Tests/TestIssues.cpp

```diff
@@ -379,6 +379,14 @@ GAME_TEST(Prs, Pr314) {
     EXPECT_EQ(pParty->pCharacters[1].GetRace(), CHARACTER_RACE_ELF);
     EXPECT_EQ(pParty->pCharacters[2].GetRace(), CHARACTER_RACE_GOBLIN);
     EXPECT_EQ(pParty->pCharacters[3].GetRace(), CHARACTER_RACE_ELF);
+
+    // test for issue 742
```

Comment:
Let's also add a proper comment to describe what exactly is being tested --- that we want to check that party qbits are set even if we press clear when creating a party